images/macos/tests/Browsers.Tests.ps1

```diff
@@ -1,49 +1,52 @@
-Describe "Chrome" {
+Import-Module "$PSScriptRoot/../helpers/Common.Helpers.psm1"
+$os = Get-OSVersion
+
+Describe "Chrome" -Skip:($os.IsVenturaArm64) {
```

Comment:
lets drop skips from `It` calls and only leave to the `Describe` blocks

Reply:
Done